scripts/stetho_open.py

```diff
@@ -78,9 +94,9 @@ def _find_only_stetho_socket(device):
   finally:
     adb.sock.close()
 
-def _connect_to_device(device=None):
+def _connect_to_device(device=None, port=None):
   adb = AdbSmartSocketClient()
-  adb.connect()
+  adb.connect(port)
```

Comment:
I was kinda figuring adb would. My python convention ignorance is showing, is parameter validation in internal methods typical?